lib/puppet/provider/acl/windows/base.rb

```diff
@@ -239,11 +236,11 @@ def are_permissions_insync?(current_permissions, specified_permissions, purge_va
           purge_value = purge_value.to_s.downcase.to_sym unless purge_value.is_a?(Symbol)
           should_purge = purge_value == :true
           remove_permissions = purge_value == :listed_permissions
-          if current_permissions.nil?
-            current_local_permissions = []
-          else
-            current_local_permissions = current_permissions.select { |p| !p.is_inherited? }
-          end
+          current_local_permissions = if current_permissions.nil?
+                                        []
+                                      else
+                                        current_permissions.reject { |p| p.inherited? }
+                                      end
```

Comment:
all this whitespace seems odd to me, but if we're cool with it...

Reply:
Yeah... it wants to align the else and end with the if.